google/ads/google_ads/v3/services/enums.py

```diff
@@ -78,7 +78,10 @@ def __getattribute__(self, name):
                 # If the class attribute has been replaced with something else,
                 # another must have already executed it and created our type.
                 return class_def_src
-            assert class_def_src.startswith('class %s(enum.' % enum_type_name), 'Expected ' + enum_type_name
+            assertion = class_def_src.startswith(
+                'class %s(enum.' % enum_type_name) or class_def_src.startswith(
+                    'class Operator(enum.')
+            assert assertion, 'Expected ' + enum_type_name
```

Comment:
Perhaps expand assertion to include both enum and operator cases?

Reply:
Do you mean separate it into two assertions and assert that either one is true?

Reply:
No, I mean to expand the assertion message to cover the `operator` case as well. The current message could be ambiguous if I understand correctly.

Reply:
Oh I see - I'm not sure that would add much, this is just asserting that the name of the first attribute on the class is of a certain pattern, either `FooBar` for the class `FooBarEnum` or simply `Operator`. In either case it's akin to an `AttributeError`.